bin/bitcoin-qt-wrapper

```diff
@@ -44,6 +45,6 @@ if ! (( RANDOM % 20 )); then
         exit 0
     fi
 fi
-cd /live/persistence/TailsData_unlocked/Persistent/.bitcoin
-space=$(( ($(df --output=avail . | tail -1) + $(du --summarize blocks | cut -f1))/1024 - 10240))
-/live/persistence/TailsData_unlocked/dotfiles/.local/bin/bitcoin-qt -dbcache=$(($(grep Available /proc/meminfo | sed s/[^0-9]//g)/1024-2100)) -prune=$(( space > 1907 ? space : 1907)) "$@"
+space=$(( ($(df --output=avail ${DATA_DIR} | tail -1) + \
```

Comment:
What do you think about if bitcoin-qt-wrapper checks whether the current prune GB setting in the qt config and/or bitcoin.conf is sufficient to not run out of space and reduces it if not?

This way is more fragile because launching bitcoin{qt,d} from the command line won't use this calculated prune setting.

Also this way the GUI prune settings won't be overridden unless the chainstate significantly grows, the user sets them too high or they save several GB of files to their Persistent Storage. Allows the user to reduce it for some reasons like backing up to smaller USB sticks or storing larger files.

I'd do the same for -dbcache but that makes these less portable, best is they instantly sync way faster when moved to a 32GB RAM machine with no other steps needed.